scripts/init.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Note that this script is supposed to be sourced
+
+# Apparently this is highly unportable and difficult to do perfectly
+# Anyhow, this could help catch some mistakes
+# http://stackoverflow.com/questions/2683279/how-to-detect-if-a-script-is-being-sourced/23009039#23009039
+not_sourced() {
+  [ "$0" = "$BASH_SOURCE" ] || [ -z "$BASH_SOURCE" ]
+}
+
+if not_sourced ; then
+  echo "Script must be sourced"
+  echo "Usage: source scripts/init.sh"
+  exit 1
+fi
+
+# A simple way to check we are in the right directory
+if [ ! -f scripts/init.sh ] ; then
+  echo "Wrong dir?  The script must be sourced starting from the project root."
+  echo "Usage: source scripts/init.sh"
+  return 2
+fi
+
+echo "Checking out the right versions of layers"
+echo poky:
+scripts/checkout_layer_hash.sh poky
+echo meta-openembedded:
+scripts/checkout_layer_hash.sh meta-openembedded
+
+source poky/oe-init-build-env
+
+echo
+echo "Now run:  bitbake miranda-image"
```

Comment:
I see this is just a comment but it says "miranda" when other scripts say "pulsar". Does this matter? Will it confuse someone (like it confuses me. :)

Reply:
This (and all other files) should not be part of PR - only ci-build.sh was intended.
I must have goofed on the last commit.  I'll fix and update.